packages/api-headless-cms-tasks/src/tasks/entries/emptyTrashBinByModelTask.ts

```diff
@@ -0,0 +1,55 @@
+import { createPrivateTaskDefinition } from "@webiny/tasks";
+import { ChildTasksCleanup } from "~/tasks/common";
+import {
+    EntriesTask,
+    HcmsTasksContext,
+    IEmptyTrashBinByModelInput,
+    IEmptyTrashBinByModelOutput
+} from "~/types";
+
+export const createEmptyTrashBinByModelTask = () => {
+    return createPrivateTaskDefinition<
+        HcmsTasksContext,
+        IEmptyTrashBinByModelInput,
+        IEmptyTrashBinByModelOutput
+    >({
+        id: EntriesTask.EmptyTrashBinByModel,
+        title: "Headless CMS - Empty trash bin by model",
```

Comment:
In the previous file, the title is written in "Title Case", but here you have it in "Sentence case".